templates/prometheus.go

```diff
@@ -45,25 +46,68 @@ var PrometheusTemplate = promv1.Prometheus{
 				Port:      intstr.FromString("web"),
 			}},
 		},
-		Resources: utils.GetResourceRequirements("prometheus"),
-		// prometheus tls -- will be uncommented
-		// Web: &promv1.WebSpec{
-		// 	TLSConfig: &promv1.WebTLSConfig{
-		// 		KeySecret: corev1.SecretKeySelector{
-		// 			Key: "tls.key",
-		// 		},
-		// 		Cert: promv1.SecretOrConfigMap{
-		// 			Secret: &corev1.SecretKeySelector{
-		// 				Key: "tls.crt",
-		// 			},
-		// 		},
-		// 		ClientAuthType: fmt.Sprint(tls.VerifyClientCertIfGiven),
-		// 		ClientCA: promv1.SecretOrConfigMap{
-		// 			ConfigMap: &corev1.ConfigMapKeySelector{
-		// 				Key: "service-ca.crt",
-		// 			},
-		// 		},
-		// 	},
-		// },
+		Resources:   utils.GetResourceRequirements("prometheus"),
+		ListenLocal: true,
+		Containers: []corev1.Container{{
+			Name: "kube-rbac-proxy",
+			Args: []string{
+				"--secure-listen-address=0.0.0.0:9339",
```

Comment:
The port number is used more than one time in this template and in the service reconciliation. 
Can we please move it into a public constant to be used in all places?